fair_mango/metrics/metrics.py

```diff
@@ -163,18 +475,54 @@ def __call__(self) -> tuple[Sequence, list]:
 
 
 class PerformanceMetric(Metric):
-    """Calculate
-    - accuracy
-    - balanced accuracy
-    - precision
-    - recall
-    - f1 score
+    """Calculate performance related metrics:
+    - accuracy.
+    - balanced accuracy.
+    - precision.
+    - recall.
+    - f1 score.
+
+    Parameters
+    ----------
+    data : Dataset | pd.DataFrame
+        Input data.
+    metrics : Collection | None, optional
```

Comment:
improve your type using Callable[blablabla]

Reply:
The typing now is `set[Callable] | dict[str, Callable] | None`